<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_goToHome">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>홈으로</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_startExercise">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>운동 시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_myData">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>분석하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_board">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>게시판</string>
    </property>
   </widget>
   <widget class="QLabel" name="tablet_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>-30</y>
      <width>1301</width>
      <height>871</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/main_frame/img/tablet_frame.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sign_img">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>1091</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/exercise/img/seleted_form.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>분석하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>560</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>▲ 분석할 부위를 선택해주세요.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ background-color: rgba(255, 255, 255, 0); }
QPushButton:hover{}
QPushButton:l:hover:pressed {}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>72</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ background-color: rgba(255, 255, 255, 0); }
QPushButton:hover{}
QPushButton:l:hover:pressed {}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_user_photo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/main/img/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/icon/img/warning.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>590</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>부위가 선택되지 않았습니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Dday_num">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Dday">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D-day</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectDay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>D-day 변경</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_profile_change">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>사진 변경</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(224, 232, 255)</string>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>가슴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>등</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>어깨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>팔</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>허벅지</string>
     </property>
    </item>
   </widget>
   <zorder>tablet_frame</zorder>
   <zorder>sign_img</zorder>
   <zorder>btn_goToHome</zorder>
   <zorder>btn_startExercise</zorder>
   <zorder>btn_myData</zorder>
   <zorder>btn_board</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>btn_logout</zorder>
   <zorder>btn_help</zorder>
   <zorder>lb_user_photo</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lb_Dday_num</zorder>
   <zorder>lb_Dday</zorder>
   <zorder>btn_selectDay</zorder>
   <zorder>btn_profile_change</zorder>
   <zorder>listWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="main_frame.qrc"/>
 </resources>
 <connections/>
</ui>
